packages/maker/flatpak/src/Config.ts

```diff
@@ -21,7 +21,7 @@ export interface MakerFlatpakOptionsConfig {
   /**
    * Release branch of the flatpak, used in the branch field of a flatpak-builder manifest.
    *
-   * Default: master
+   * Default: main
```

Comment:
I think these aren't to do with the `master` branch on this repo, so probably shouldn't be changed in this PR.